server/server.ts

```diff
@@ -23,28 +24,26 @@ const port = env.PORT;
 const app = nextApp({ dir: "./client", dev: env.isDev });
 const handle = app.getRequestHandler();
 
+Bugsnag.start({
+  apiKey: env.BUGSNAG_API_KEY,
+  plugins: [BugsnagPluginExpress],
+  releaseStage: env.BUGSNAG_RELEASE_STAGE
+})
+
 app.prepare().then(async () => {
   const server = express();
+  var middleware = Bugsnag.getPlugin('express')
+
+  // Bugsnag: Capture errors in downstream middleware
+  server.use(middleware.requestHandler)
 
   server.set("trust proxy", true);
 
   if (env.isDev) {
     server.use(morgan("combined", { stream }));
-  } else if (env.SENTRY_PRIVATE_DSN) {
-    Sentry.init({
-      dsn: env.SENTRY_PRIVATE_DSN,
-      environment: process.env.NODE_ENV
-    });
-
-    server.use(
-      Sentry.Handlers.requestHandler({
-        ip: true,
-        user: ["id", "email"]
-      })
-    );
   }
 
-  server.use(helmet());
+  server.use(helmet({ contentSecurityPolicy: false }));
```

Comment:
did this come in through the merge?

Reply:
Assume so. I didn't change that.

Reply:
hmm, its not something we should be disabling, are you able to test that without disabling that policy?

Reply:
sounds wise, yep - will do 👍

Reply:
Seems to test fine with contentSecurityPolicy set to true